proxy/grpcproxy/cluster.go

```diff
@@ -47,6 +126,10 @@ func (cp *clusterProxy) MemberUpdate(ctx context.Context, r *pb.MemberUpdateRequ
 }
 
 func (cp *clusterProxy) MemberList(ctx context.Context, r *pb.MemberListRequest) (*pb.MemberListResponse, error) {
+	if cp.wa != nil {
+		// fake the member lists with grpc-proxy members
+		return &pb.MemberListResponse{Members: cp.members()}, nil
+	}
 	conn := cp.client.ActiveConnection()
```

Comment:
this (and the other RPCs) should use the `client.Cluster` calls so the proxy can be stacked